ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -2131,7 +2131,7 @@ public sealed class ArchiWebHandler : IDisposable {
 				return null;
 			}
 
-			const string requestValidateCode = "/account/validatewalletcode";
+			const string requestValidateCode = "/account/ajaxredeemwalletcode/";
```

Comment:
Is that trailing slash really needed? I prefer to have possibly the shortest URLs, and until now everything that expected it also worked without it.

Reply:
Looks like it's not needed, will remove it now.